### PR TITLE
Add public AGS page

### PR DESCRIPTION
**Pull request by brmscheiner**
Pull request checklist:
 - [x] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
 - [x] All text is internationalized
 - [ ] There are no linter errors
 Only linter errors are the unresolved "no-unstable-nested-components" issue for customBodyRender in tables. Still dont have a good solution for this 
 - [x] New features support all states (loading, error, etc) 

Thanks for keeping it clean! Feel free to merge your own pull request after it has been approved - just use the "squash & merge" option.

### CODE REVIEW

**Comment by brmscheiner** [comment]
image

File: src/components/dataDisplays/HoustonSightingsDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```
Comment (brmscheiner) [inline src/components/dataDisplays/HoustonSightingsDisplay.jsx:1]: This is set up to be used my `SightingsCard`. I am trying to do separate PRs so the feature doesn't get slowed down by the cleanup

**Review by Atticus29 — COMMENTED** [review]
A few comments.

File: src/pages/assetGroup/ReviewPendingPublicSightings.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  if (loading) return <LoadingScreen />;
+
+  return (
+    <MainColumn fullWidth>
+      <EntityHeader
+        name={intl.formatMessage({
+          id: 'REVIEW_PENDING_PUBLIC_SIGHTINGS',
+        })}
+      />
+      <HoustonSightingsDisplay
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/pages/assetGroup/ReviewPendingPublicSightings.jsx:38]: that is okay - delete should not be a primary action here. they should at least be clicking on a sighting and seeing what's in there before they delete it

File: src/components/AuthenticatedAppHeader/ActionsPane.jsx [hunk 3]
```diff
@@ -29,10 +30,19 @@ const actions = [
     messageId: 'SETTINGS_AND_PRIVACY',
     icon: SettingsIcon,
   },
+  {
+    id: 'review-pending-public-sightings',
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/components/AuthenticatedAppHeader/ActionsPane.jsx:34]: sounds good to me

File: src/components/AuthenticatedAppHeader/ActionsPane.jsx [hunk 4]
```diff
@@ -29,10 +30,19 @@ const actions = [
...
+      userData?.is_admin || userData?.is_data_manager,
+    messageId: 'PENDING_PUBLIC_SIGHTINGS',
+    icon: PublicIcon,
+  },
   {
     id: 'control-panel',
     href: '/admin',
-    adminOrUserManagerOnly: true,
+    permissionsTest: userData =>
+      userData?.is_admin || userData?.is_user_manager,
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/components/AuthenticatedAppHeader/ActionsPane.jsx:45]: yeah, if there was a property to directly check whether this action was available i would use it. but for now i think this is the best we can do

**Review by Atticus29 — APPROVED** [review]
New changes look good. Other than Emily's comment about bulk import, this seems fine to me!

**Comment by brmscheiner** [comment]
@Emily-Ke no, we expect logged in users to process their own reported sightings. this table is meant to give administrators and data managers the ability to process anonymously reported sightings

**Comment by brmscheiner** [comment]
I wasn't aware of that, we may need another ticket to do something with contributor AGS. As DEX-1129 only covers displaying public AGS, I think this PR addresses the original ticket

**Review by Emily-Ke — DISMISSED** [review]
❗ Please confirm that this doesn't break bulk import:
https://github.com/WildMeOrg/codex-frontend/pull/416#discussion_r915282785

File: src/AuthenticatedSwitch.jsx [hunk 5]
```diff
@@ -20,6 +20,7 @@ import Sighting from './pages/sighting/Sighting';
 import AssetGroupSighting from './pages/sighting/AssetGroupSighting';
 import Splash from './pages/splash/Splash';
 import AssetGroup from './pages/assetGroup/AssetGroup';
+import ReviewPendingPublicSightings from './pages/assetGroup/ReviewPendingPublicSightings';
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/AuthenticatedSwitch.jsx:23]: do you want me to move it to a different folder?

File: src/AuthenticatedSwitch.jsx [hunk 6]
```diff
@@ -150,6 +151,9 @@ export default function AuthenticatedSwitch({
                       <Route path="/notifications">
                         <Notifications />
                       </Route>
+                      <Route path="/pending-public-sightings">
+                        <ReviewPendingPublicSightings />
+                      </Route>
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/AuthenticatedSwitch.jsx:156]: we could use that url as well. would you like me to change it?

File: src/AuthenticatedSwitch.jsx [hunk 7]
```diff
@@ -150,6 +151,9 @@ export default function AuthenticatedSwitch({
                       <Route path="/notifications">
                         <Notifications />
                       </Route>
+                      <Route path="/pending-public-sightings">
+                        <ReviewPendingPublicSightings />
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/AuthenticatedSwitch.jsx:155]: i can rename it for you

File: locale/en.json [hunk 8]
```diff
@@ -12,6 +12,8 @@
   "LABEL": "Label",
   "TYPE": "Type",
   "FOREST": "Forest",
+  "STAGE": "Stage",
+  "NO_PENDING_PUBLIC_SIGHTINGS": "There are no pending public sightings",
```
Comment (Emily-Ke) [inline locale/en.json:16]: Does this need to be changed too?

File: src/components/dataDisplays/HoustonSightingsDisplay.jsx [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+      name: 'created',
+      labelId: 'DATE_REPORTED',
+      options: {
+        cellRenderer: cellRendererTypes.date,
+      },
+    },
+    {
+      reference: 'submissionTime',
+      name: 'submissionTime',
+      labelId: 'DATE_REPORTED',
```
Comment (Emily-Ke) [inline src/components/dataDisplays/HoustonSightingsDisplay.jsx:49]: Are two columns with the same `labelId` assumed to never be used at the same time?

Comment (brmscheiner) [reply src/components/dataDisplays/HoustonSightingsDisplay.jsx:49]: correct. one or the other

File: locale/en.json (hunk 8 above)
Comment (brmscheiner) [reply locale/en.json:16]: i left it for now so that it was clearer why the table was empty. we can adjust when contributor sightings are there as well to say something like "there are no pending public or contributor sightings"
